.github/workflows/build-main.yml

```diff
@@ -0,0 +1,66 @@
+name: Build Main/Master SNAPSHOT
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "5 6 * * *"
```

Comment:
Personally, I think a nightly build is unnecessary if we're building on every push. Seems like a waste of resources to me.

Reply:
Easy to remove if you'd like